### PR TITLE
Remove "or refreshes" example language from `overscroll-behavior: contain` docs

### LINKED ISSUE #29473: `overscroll-behavior: contain` example specifies "or refreshes" when it shouldn't
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/overscroll-behavior#values

### What specific section or headline is this issue about?

Values

### What information was incorrect, unhelpful, or incomplete?

The doc says:
> Default scroll overflow behavior is observed inside the element this value is set on (e.g. "bounce" effects **or refreshes**)...

The "or refreshes" part shouldn't be there, because native browser navigation isn't allowed when `overscroll-behavior: contain` is set.

### What did you expect to see?

> Default scroll overflow behavior is observed inside the element this value is set on (e.g. "bounce" effects)...

### Do you have any supporting links, references, or citations?

This issue was borne from a conversation with @dlrobertson of the Firefox Layout/Asynchronous Panning and Zooming (APZ) team, who said:

> - The issue had to do with the `overscroll-behavior` docs
> - specifically the `contain` docs say `... (e.g. "bounce" effects or refreshes)`
> - [The spec for contain](https://drafts.csswg.org/css-overscroll/#valdef-overscroll-behavior-contain) indicates all [non-local-boundary-default-action](https://drafts.csswg.org/css-overscroll/#non-local-boundary-default-action) (including navigation) are not allowed
> - Note the spec example [here](https://drafts.csswg.org/css-overscroll/#example-64e64c20) indicates "A page wants to implement their own pull-to-refresh effect when `contain` is used"
> - So simply "or refreshes" should probably be removed from the MDN docs for `overscroll-behavior: contain`
> - That being said browser implementations are a bit all over the place here... We don't ship pull to refresh to the release channel and there are some implementation bugs out there
> - Botond Ballo, the tech lead of APZ discovered this... He's real helpful and knowledgeable on APZ things

### Do you have anything more you want to share?

This is my first submitted issue to MDN 🎉 yay

### PR DESCRIPTION
**Pull request by zfox23**
### Description

The current doc says:
> Default scroll overflow behavior is observed inside the element this value is set on (e.g. "bounce" effects **or refreshes**)...

The "or refreshes" part shouldn't be there, because native browser navigation isn't allowed when `overscroll-behavior: contain` is set.

### Motivation

This removes misleading information from `overscroll-behavior` documentation.

### Related issues and pull requests

Resolves: https://github.com/mdn/content/issues/29473 (there are more details in that issue)

### CODE REVIEW

File: files/en-us/web/css/overscroll-behavior-block/index.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ The `overscroll-behavior-block` property is specified as a keyword chosen from t
 - `auto`
   - : The default scroll overflow behavior occurs as normal.
 - `contain`
-  - : Default scroll overflow behavior is observed inside the element this value is set on (e.g. "bounce" effects or refreshes), but no scroll chaining occurs to neighboring scrolling areas, e.g. underlying elements will not scroll.
+  - : Default scroll overflow behavior is observed inside the element this value is set on (e.g. "bounce" effects), but no scroll chaining occurs to neighboring scrolling areas, e.g. underlying elements will not scroll.
```
Comment (estelle) [inline files/en-us/web/css/overscroll-behavior-block/index.md:37]: Should we add that `contain` disables pull-to-refresh while allowing swipe navigation, preventing a user's over-scrolling triggering navigation?

Comment (theres-waldo) [reply files/en-us/web/css/overscroll-behavior-block/index.md:37]: `contain` disables both pull-to-refresh and swipe navigation.

The reason [this spec example](https://drafts.csswg.org/css-overscroll/#example-64e64c20) says "only disable pull-to-refresh but allow swipe navigations" is that the property being set is `overscroll-behavior-y: contain;`, and swipe navigation is a horizontal gesture. So `overscroll-behavior-y: contain` will disable pull to refresh (which is a vertical gesture), which `overscroll-behavior-x` will remain at its default value of `auto` which allows swipe navigation.

Comment (dipikabh) [reply files/en-us/web/css/overscroll-behavior-block/index.md:37]: Can we rephrase this a bit for more clarity? Also to be more specific, how about adding the bit about "pull-to-refresh" and "swipe navigation"? And if updating here, the same changes can be applied to other files in this PR.

```suggestion
  - : Default scroll overflow behavior (e.g., "bounce" effects) is observed inside the element where this value is set. However, no scroll chaining occurs on neighboring scrolling areas; for example, the underlying elements will not scroll. The `contain` value disables both pull-to-refresh and swipe navigation.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks @zfox23 for opening the PR! LGTM. I've suggested some edit.

File: files/en-us/web/css/overscroll-behavior-block/index.md (hunk 1 above)
Comment (zfox23) [reply files/en-us/web/css/overscroll-behavior-block/index.md:37]: Thank you all for the suggestions! It was super helpful for me to see how folks jumped in to improve this PR.

I've made another commit in an effort to address your comments. Please let me know if you approve of these changes.

**Review by dipikabh — APPROVED** [review]
LGTM. Thanks for improving these pages, @zfox23!

**Pull request merged**